<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientwidget</class>
 <widget class="QWidget" name="clientwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>160</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Camera</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="camera_open_button">
        <property name="text">
         <string>On</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="camera_close_button">
        <property name="text">
         <string>Off</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Microphone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="microphone_open_button">
          <property name="text">
           <string>On</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="microphone_close_button">
          <property name="text">
           <string>Off</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>160</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPushButton" name="client_exit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Room members</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="refresh_button">
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="client_display"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="chat_display">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>266</y>
     <width>571</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Message history</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="chat_enter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="camera_show">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>24</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
